spec/requests/citizens/check_answers_spec.rb

```diff
@@ -68,7 +68,6 @@
     context 'applicant does not own home' do
       let(:legal_aid_application) { create :legal_aid_application, :with_everything, :without_own_home }
       it 'does not display property value' do
-        expect(response.body).not_to include(number_to_currency(legal_aid_application.property_value, unit: '£'))
```

Comment:
what was wrong with these?

Reply:
The variance of values is not great enough. That is, occasionally there is another value on the page that has the same amount. So you get a false failure. The spec already checks the section isn't there. By adding a check that NOWHERE ON THE PAGE is there the same value, the spec is just being made more likely to cause an erroneous result.